$OpenBSs ~

dont check for SQLITE_SECURE_DELETE at runtime, we disable it

Index: storage/mozStorageService.cpp
--- storage/mozStorageService.cpp.orig
+++ storage/mozStorageService.cpp
@@ -188,7 +188,6 @@ already_AddRefed<Service> Service::getSingleton() {
   // or newer.  Our configure check ensures we are using a new enough version
   // at compile time.
   if (SQLITE_VERSION_NUMBER > ::sqlite3_libversion_number() ||
-      !::sqlite3_compileoption_used("SQLITE_SECURE_DELETE") ||
       !::sqlite3_compileoption_used("SQLITE_THREADSAFE=1") ||
       !::sqlite3_compileoption_used("SQLITE_ENABLE_FTS3") ||
       !::sqlite3_compileoption_used("SQLITE_ENABLE_UNLOCK_NOTIFY") ||
